src/test/java/org/symphonyoss/symphony/messageml/elements/ExpandableCardTest.java

```diff
@@ -0,0 +1,170 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.Test;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+
+import java.util.Collections;
+
+public class ExpandableCardTest extends ElementTest {
+
+  private void verifyExpandableCard(Element card) throws Exception {
```

Comment:
Private methods below public ones.

Reply:
Hi @lmacedodaitan I changed it (also in the source class where I copy-pasted...)